apps/bluetooth/js/modules/transfer_manager.js

```diff
@@ -175,7 +175,8 @@ define(function(require) {
     _onBluetoothEnabledChanged:
     function bttm__onBluetoothEnabledChanged(enabled) {
       Debug('_onBluetoothEnabledChanged(): enabled = ' + enabled);
-      if (enabled) {
+      // make sure the value is defined
```

Comment:
The origin intention is to align with other observe function call, but it's fine to just not running the "changed" callback at all initially. So I'll remove it.